src/icrar/leap-accelerate/tests/model/MetaDataTests.cc

```diff
@@ -188,6 +188,7 @@ namespace icrar
             EXPECT_DOUBLE_EQ(2.1537588131757608, meta.GetConstants().GetChannelWavelength(0));
         }
 
+#ifdef CUDA_ENABLED
```

Comment:
Same comment regarding rutime v/s compile time check, same for the rest of the file.

Reply:
Again, I was too quick here and misjudged, there is code that needs to be protected by the macro.